.github/workflows/main.yaml

```diff
@@ -22,8 +23,11 @@ jobs:
         with:
           node-version: '14'
       - name: Build and publish clients
-        if: ${{ steps.vars.outputs.has-changes }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        if: '${{ steps.vars.outputs.has-changes }}'
         run: |
           sudo apt-get install moreutils
+          echo "//npm.pkg.github.com/:_authToken=${{ secrets.GITHUB_TOKEN }}" > ~/.npmrc
```

Comment:
echo "//npm.pkg.github.com/:_authToken=${{ secrets.GITHUB_TOKEN }}" >> ~/.npmrc

Reply:
Fixed in latest commit

Reply:
I think `>` is more idempotent? E.g.

```
➜  ~ echo "something" >>file
➜  ~ echo "something" >>file
➜  ~ echo "something" >>file
➜  ~ echo "something" >file 
➜  ~ echo "something" >file
➜  ~ echo "something" >file
```

Reply:
`>file ` overwrites a file, `>>file` appends to the file. Since we have an `.npmrc` file, we don't want to overwrite the content.

Reply:
Yes I know, but I wonder what the use is of appending this line indefinitely to that file? Just overwrite it every time to resolve this url?